### PR TITLE
fix: Giraffe gauge

### LINKED ISSUE #629: Replace Gauge graph in UI with Gauge from Giraffe
the goal of this is to replace the gauge implementation in the UI with the implementation in Giraffe lib.

This would allow us to improve the graph without making the change in two places.

### PR DESCRIPTION
**Pull request by zoesteinkamp**
Closes #629

We are changing the gauge graph to reference the giraffe gauge graph
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass

### CODE REVIEW

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 40 | :red_circle:
accessibility | 87 | :green_circle:
best-practices | 87 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :question:

**Comment by zoesteinkamp** [comment]
now we just need a feature flag!

File: src/visualization/types/Gauge/view.tsx [hunk 1]
```diff
@@ -29,32 +29,52 @@ const GaugeChart: FC<Props> = ({result, properties}) => {
...
+          suffix: suffix,
+          tickPrefix: tickPrefix,
+          tickSuffix: tickSuffix,
+          decimalPlaces: decimalPlaces,
+          gaugeColors: colors,
+        },
+      ],
+    }
+    return <Plot config={config} />
+  } else {
```
Comment (hoorayimhelping) [inline src/visualization/types/Gauge/view.tsx:52]: since this returns, you don't even need the `else` here

**Review by hoorayimhelping — APPROVED** [review]

**Review by hoorayimhelping — COMMENTED** [review]
This LGTM, but I think we should wait for Timmy

File: src/visualization/types/Gauge/view.tsx (hunk 1 above)
Comment (TCL735) [reply src/visualization/types/Gauge/view.tsx:52]: i agree with this. imo, dropping the else and putting the return outside looks better.

**Review by TCL735 — APPROVED** [review]

**Review by ChitlangeSahas — APPROVED** [review]

**Pull request merged**